packages/plugin-core/src/constants.ts

```diff
@@ -58,6 +58,11 @@ export const DENDRON_VIEWS = [
     where: "explorer",
     icon: "media/icons/dendron-vscode.svg",
   },
+  {
+    ...treeViewConfig2VSCodeEntry(DendronTreeViewKey.LOOKUP_VIEW),
+    when: `${DendronContext.WEB_UI_ENABLED} && ${DendronContext.NOTE_LOOK_UP_ACTIVE}`,
```

Comment:
we aren't using webUIEnabled flag any more so you can remove it

Reply:
It looks like the context is still being set and `enableWebUI` config still exists too. It looks like treeview v2 is using it for enablement.

Reply:
yep. its a binary flag for tree view v2. i meant we don't use it for new web views any more so you can remove it from the lookup view